api/controllers/UserController.js

```diff
@@ -234,6 +235,318 @@ module.exports = {
     });
   },
 
+  permaBan: function (req, res) {
+    /*  Form parameters:
```

Comment:
Nice, I like this. This sort of thing is massively lacking in the app, comments that explain things :-P